csrankings-a.csv

```diff
@@ -86,6 +86,7 @@ Aarti Singh,Carnegie Mellon University,http://www.cs.cmu.edu/~aarti,6jP3ZSEAAAAJ
 Aasa Feragen,DTU,http://www2.compute.dtu.dk/~afhar,MNDVpoUAAAAJ
 Aasheesh Kolli,Pennsylvania State University,https://aasheeshkolli.com,UcfzOQYAAAAJ
 Aastha Mehta,University of British Columbia,https://aasthakm.github.io,l1jObzoAAAAJ
+Aiping Xiong,Pennsylvania State University,https://ist.psu.edu/directory/axx29,sJ-tJogAAAAJ

```

Comment:
Not in EECS. https://www.eecs.psu.edu/departments/cse-faculty-list.aspx

Reply:
Hi Emery,

At Penn State, the distinction between EECS and IST (Information Science and Technology) is primarily symbolic. This is because most IST faculty hold PhDs in Computer Science and conduct research in sub areas of the field, including AI, Human-Computer Interaction, Cyber Security, AI Ethics, and Social Informatics.

Many of our faculty members are featured on csrankings.org.

For reference, I am citing a few:

Aron Laszka (https://ist.psu.edu/directory/aql5923)
Sharon X. Huang (https://ist.psu.edu/directory/suh972)
Shomir Wilson (https://ist.psu.edu/directory/suw724)
Hadi Hosseini (https://ist.psu.edu/directory/hbh5279)
John M. Carroll (https://ist.psu.edu/directory/jmc56)
In my pull request, I included those with relevant publications in Human-Computer Interaction.

You can view the complete list of IST faculty here: https://ist.psu.edu/directory?field_dir_1stletter_lname_value=All&field_dir_affiliation_target_id_1=31&field_dir_staffunit_target_id=All&field_dir_facresearchtags_target_id=All&title=